File: sample/src/main/java/com/ebnbin/ebapplication/sample/MainActivity.java
package com.ebnbin.ebapplication.sample;

import android.content.Intent;
import android.os.Bundle;
import android.support.annotation.Nullable;
import android.view.View;
import android.widget.Button;

import com.ebnbin.ebapplication.context.ui.EBActivity;

public final class MainActivity extends EBActivity {
    @Override
    protected void onCreate(@Nullable Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);

        Button button = new Button(this);

        setContentView(button);

        button.setOnClickListener(new View.OnClickListener() {
            @Override
            public void onClick(View v) {
                startActivity(new Intent(MainActivity.this, MainActivity.class));
            }
        });
    }
}

File: sample/src/main/java/com/ebnbin/ebapplication/sample/MainActivity.kt
package com.ebnbin.ebapplication.sample

import android.os.Bundle
import com.ebnbin.ebapplication.context.EBActivity

class MainActivity : EBActivity() {
    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)

        fragmentHelper.set(TestActionBarFragment())
    }
}
